core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -678,7 +678,7 @@ class UnifiedLog(@volatile var logStartOffset: Long,
    *
    * @throws KafkaStorageException if rename fails
    */
-  def renameDir(name: String): Unit = {
+  def renameDir(name: String, reinitialize: Boolean): Unit = {
```

Comment:
can we write some comments to describe what `reinitialize` means and in what case it should be set to true/false?

Reply:
Thanks for the review. I've added some doc to explain this parameter.